src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerSession.cs

```diff
@@ -20,6 +20,7 @@ public class ProfilerSession
         private static readonly TimeSpan DefaultPollingDelay = TimeSpan.FromSeconds(1);
         private object pollingLock = new object();
         private bool isPolling = false;
+        public bool isStreaming = false;
```

Comment:
this should be private and use property functions if it really needs to be public.  if we're adding this variable do we still need `isPolling` and polling variables below?  the new variable seems to be replacing the existing one?

Reply:
@smartguest I missed seeing this comment from Karl. So you can put the getter/setter back in if that makes sense to you (you can just use the auto-getter and setter `public bool IsStreaming { get; set; }` to make that easy)